spec/features/court_applications_spec.rb

```diff
@@ -26,6 +26,23 @@
     expect(page).to have_content "There was a problem getting the information you requeste"
   end
 
+  context 'when the FF is enabled' do
+    before do
+      allow(FeatureFlag).to receive(:enabled?).with(:maintenance_mode).and_return(false)
```

Comment:
I thought this was maintenance_mode was false by default

Reply:
It is, I was trying to deal with RSpec complaining if an `allow`ed method gets called with different arguments, but I realised I can do it better. Now updated.